### DISCUSSION TITLE
node-valkey

### DISCUSSION THREAD

**Discussion by FredM7** [valkey-io/valkey#332] [Q&A]
Super excited about this project. (Not sure if this should be an `Idea`, but used `Q&A` category)

Are there plans to create, eg: `node-valkey` similar to https://github.com/redis/node-redis?

**Comment by stockholmux** [comment]
My PoV is that if a client library works right now, then there isn't a need to create a Valkey version. @FredM7 is there something particularly that you're not seeing working or something you want that is different?

**Comment by zuiderkwast** [comment]
We discussed forking clients in the last core team meeting. Here are the notes: https://github.com/orgs/valkey-io/discussions/362

The main point IMO is that someone needs to do the work. It will only happen if we have a nodejs developer willing to maintain such fork.

**Reply by zuiderkwast** [reply]
See this: https://github.com/valkey-io/valkey/issues/329

**Reply by FredM7** [reply]
Makes sense, thanks 👍🏻
